src/tests/e2e/feature/main/main.cy.ts

```diff
@@ -0,0 +1,38 @@
+export {}
+
+const resultPage = (): void => {
```

Comment:
```suggestion
const visitResultPage = (): void => {
```
페이지 접속하는 네이밍으로 개선하는 것도 괜찮을 것 같은데 어떠신가요?

Reply:
넵 이게 훨씬 좋을 것 같습니다!

Reply:
result page가 따로있어서 요기는 main page 아잉가용 ?